python/examples/advanced/String_Tracking.ipynb

```diff
@@ -0,0 +1,767 @@
+       "0         12     Alice           my_email_1989@gmail.com\n",
+       "1         83       Bob                          ADK-1171\n",
+       "2          1   Chelsea   Copacabana 272 - Rio de Janeiro\n",
+       "3        992     Danny  21º C Friday - Sao Paulo, Brasil\n",
+       "4          7     Eddie                       18127819ASW"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import whylogs as why\n",
+    "import pandas as pd\n",
+    "data = {\n",
+    "    \"onlyDigits\": [\"12\", \"83\", \"1\", \"992\", \"7\"],\n",
+    "    \"onlyAlpha\": [\"Alice\", \"Bob\", \"Chelsea\", \"Danny\", \"Eddie\"],\n",
+    "    \"mixed\": [\"my_email_1989@gmail.com\",\"ADK-1171\",\"Copacabana 272 - Rio de Janeiro\",\"21º C Friday - Sao Paulo, Brasil\",\"18127819ASW\"]\n",
+    "}\n",
+    "df = pd.DataFrame(data)\n",
+    "df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Configuring the Metrics in the DatasetSchema"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "whylogs uses `Resolvers` in order to define how a column name or data type gets mapped to different metrics.\n",
+    "In this case, we'll create a custom Resolver to apply the UnicodeRangeMetric to all of the columns.\n",
+    "\n",
+    "> If you're interested in seeing how you can add or remove different metrics according to the column type or column name, please refer to this example on [Schema Configuration](https://whylogs.readthedocs.io/en/stable/examples/basic/Schema_Configuration.html)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from whylogs.core.schema import ColumnSchema, DatasetSchema\n",
+    "from whylogs.core.metrics.unicode_range import UnicodeRangeMetric\n",
+    "from whylogs.core.resolvers import Resolver\n",
+    "from whylogs.core.datatypes import DataType\n",
+    "from typing import Dict\n",
+    "from whylogs.core.metrics import Metric, MetricConfig\n",
+    "\n",
+    "class UnicodeResolver(Resolver):\n",
+    "    def resolve(self, name: str, why_type: DataType, column_schema: ColumnSchema) -> Dict[str, Metric]:\n",
+    "        return {UnicodeRangeMetric.get_namespace(column_schema.cfg): UnicodeRangeMetric.zero(column_schema.cfg)}\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Resolvers are passed to whylogs through a `Dataset Schema`, so we'll have to create a custom Schema as well.\n",
+    "\n",
+    "We will create a custom Schema that inherits from DatasetSchema. We'll just have to:\n",
+    "- Pass the UnicodeResolver created previously\n",
+    "- Since we're interested in a default range, we'll also pass a default Metric Configuration with the desired ranges"
```

Comment:
"interested in changing the default character ranges"

Reply:
done!